source/developer/fx-guidelines.rst

```diff
@@ -242,134 +351,113 @@ Input: Hints
 
-    Correct:
-        When there are two buttons on bottom right, left button should be “Cancel” and the button on the right should be the [ACTION_BUTTON], in this case “Send Invitations”.
+      ..  image:: ../images/confirm2.jpg
+          :alt: Settings saved
 
-        ..  image:: ../images/buttonPlacement2.png
-            :alt: Button Placement 2
-            :width: 500 px
+  Example 2:
+    "Link copied to clipboard"
 
-    Incorrect:
-        When there are two buttons at the bottom right, left button should not be “Close”, as it’s not clear if closing will or won’t execute the [ACTION_BUTTON].
+      ..  image:: ../images/confirm1.jpg
+          :alt: Confirmation message
 
-        ..  image:: ../images/buttonPlacement3.png
-            :alt: Button Placement 3
-            :width: 500 px
+**Save prompt:**
 
------
+A prompt should appears if a user makes changes to a setting and attempts to navigate away without saving them.
 
-Display: Number of Choices
---------------------------
+  Example:
+    "You have unsaved changes, are you sure you want to discard them?"
 
-To simplify decisions, when practical, limit the number of choices to 3 or 4, and add separators or headings between logical groups. See `Hick’s Law <https://en.wikipedia.org/wiki/Hick%27s_law>`_ for background on why this helps.
+      ..  image:: ../images/save1.png
+          :alt: Save prompt
 
-Example:
+---------------------------
 
-    Incorrect:
-        No clear separation between distinct options.
+Menu items
+----------
 
-        ..  image:: ../images/choices1.png
-            :alt: No separation
-            :width: 500 px
+Items in a menu should be grouped together based on the category they belong to. Items per category however should not exceed 5 for the app and 7 for system console.
 
-    Correct:
-        A clear separation between distinct options.
+**Example:**
 
-        ..  image:: ../images/choices2.png
-            :alt: Clear separation
-            :width: 500 px
+    ..  image:: ../images/group.png
+        :alt: Center channel post area
 
 -----
 
-Display: Alignment
-------------------
-
-**Elements should feature margins horizontally and vertically, evenly spaced.**
-
-Create space between elements, such as buttons, text, line separators, headers and backgrounds, by leaving even space around them (either equal space or at most 1 pixel difference).
-
-    **Example:**
+Notifications: Header
+---------------------
 
-        Button positioned in the middle of the header.
-            ..  image:: ../images/align1.jpg
-                :alt: Button positioning
-                :width: 500 px
+Appears as an additional header at the top of the UI. May include an “x” button to dismiss. Limited to 60 characters. May include a time-out. Header notification should appear over the rest of the UI instead of pushing the UI down. Multiple headers can appear, with the earliest notification at the top and more recent notifications appearing underneath.
```

Comment:
Propose " The header bar appears as a dropdown bar from the top of the screen. The header bar:

1. May include an "x" button to dismiss
2. Be limited to 60 characters
3. May include a time-out
4. Appear over the rest of the UI instead of pushing the UI down

Multiple headers can appear, with the earliest notification at the top and more recent notifications appearing underneath.